### DISCUSSION TITLE
User credentials prompt (what is?)

### DISCUSSION THREAD

**Discussion by iceblue1978** [ChrisTitusTech/winutil#2734] [Q&A]
Hi! When i try to 'get installed' and 'install selected', i get this 'user credentials prompt' for some app and don't know what to put there....if i skip, winutil will stuck and must be closed.

**Comment by NeoFax75** [comment]
Same here, with no response.

**Comment by iceblue1978** [comment]
Waiting for an answer from [ChrisTitusTech](https://github.com/ChrisTitusTech/ChrisTitusTech) :)
